src/internal/components/ThemeEditor.tsx

```diff
@@ -95,9 +96,11 @@ export const ThemeEditor = (props: ThemeEditorProps) => {
       if (localHistoryArray) {
         devLogger.log("Theme Dev Mode - Using theme localStorage");
         const localHistories = JSON.parse(localHistoryArray);
-        const localHistoryIndex = localHistories.length - 1;
+        const localHistoryIndex = JSON.parse(localHistoryArray).findIndex(
```

Comment:
devMode doesn't use saveState. This should just set the index to the last item just like loadPuckInitialHistory